websites/T/TrueID/presence.ts

```diff
@@ -0,0 +1,166 @@
+            ? `${
+                privacy
+                  ? ""
+                  : lang === "th-th"
+                  ? `ตอนที่ ${ep[0]}`
+                  : `Ep ${ep[0]}`
+              }`
+            : ""
+        }`;
+        presenceData.state = `${result_[1] ? result_[1] : result}`;
+        if (buttons) {
+          presenceData.buttons = [
+            {
+              label: `${lang === "th-th" ? "ดูซีรีส์" : "Watch Series"}`,
+              url: document.location.href.replace(/#\d+/, "")
+            }
+          ];
+        }
+        break;
+      case path.pathname.includes("movie"):
+        presenceData.details = `${lang === "th-th" ? "หนัง" : "Movie"}`;
+        presenceData.state = `${title}`;
+        if (buttons) {
+          presenceData.buttons = [
+            {
+              label: `${lang === "th-th" ? "ดูหนัง" : "Watch Movies"}`,
+              url: document.location.href.replace(/#\d+/, "")
+            }
+          ];
+        }
+        break;
+      default:
+        presenceData.details = "Movie";
+        presenceData.state = lang === "th-th" ? "หน้าแรก" : "Home page";
+        break;
+    }
+  } else if (path.hostname === "tv.trueid.net") {
+    const title =
+        document.querySelector(
+          "#__next > div > div.wrapper.-spacing > div.player-block.d-flex > div > div.desc-nowrap.d-flex > div.desc-block.title-middle > div > h1"
+        )?.textContent ?? "",
+      result = title.replace("ดูช่อง ", "").replace("ออนไลน์", "").split("–");
+    switch (true) {
+      case path.pathname.includes("live"):
+        presenceData.details = `${
+          lang === "th-th" ? "ดูทีวีออนไลน์" : "TV Online"
+        }`;
+        presenceData.state = `${lang === "th-th" ? "ช่อง" : "Channel"} ${
+          result[0]
+        }`;
+        presenceData.smallImageKey = "live";
+        if (buttons) {
+          presenceData.buttons = [
+            {
+              label: `${
+                lang === "th-th" ? "ดูทีวีออนไลน์" : "Watch tv online"
+              }`,
+              url: document.location.href.replace(/#\d+/, "")
+            }
+          ];
+        }
+        break;
+      case path.pathname.includes("tvguide"):
+        presenceData.details = "Tv Guide";
+        presenceData.state =
+          lang === "th-th"
+            ? "ผังรายการทีวีช่องทีวีทั้งหมด"
+            : "Program schedule of all channels";
+        presenceData.smallImageKey = "question";
+        break;
+      default:
+        presenceData.details = "Tv";
+        presenceData.state = lang === "th-th" ? "หน้าแรก" : "Home page";
+        break;
+    }
+  } else {
+    delete presenceData.startTimestamp;
+    delete presenceData.endTimestamp;
+  }
+  if (!time) {
+    delete presenceData.startTimestamp;
+    delete presenceData.endTimestamp;
+  }
+  if (privacy) {
+    delete presenceData.state;
+    delete presenceData.buttons;
+  }
+  if (!presenceData.details) presence.setActivity();
+  else presence.setActivity(presenceData);
```

Comment:
#4946 will enforce all presences to use this format
```suggestion
  if (presenceData.details) presence.setActivity(presenceData);
  else presence.setActivity();
```